cloudinit/util.py

```diff
@@ -551,7 +551,9 @@ def system_info():
         elif linux_dist == 'redhat':
             var = 'rhel'
         elif linux_dist in (
-                'opensuse', 'opensuse-tumbleweed', 'opensuse-leap', 'sles'):
+                'opensuse', 'opensuse-tumbleweed', 'opensuse-leap',
+                 'sles', 'sle_hpc'
+        ):
```

Comment:
flake8 doesn't like the formatting
```
(crispyboi) cloud-init % git diff
diff --git a/cloudinit/util.py b/cloudinit/util.py
index b6f1117f..6c1675b7 100644
--- a/cloudinit/util.py
+++ b/cloudinit/util.py
@@ -551,7 +551,9 @@ def system_info():
         elif linux_dist == 'redhat':
             var = 'rhel'
         elif linux_dist in (
-                'opensuse', 'opensuse-tumbleweed', 'opensuse-leap', 'sles'):
+                'opensuse', 'opensuse-tumbleweed', 'opensuse-leap',
+                 'sles', 'sle_hpc'
+        ):
             var = 'suse'
         else:
             var = 'linux'
(crispyboi) cloud-init % python3 -m flake8 cloudinit/util.py
cloudinit/util.py:555:18: E131 continuation line unaligned for hanging indent
     ..:(╯°□°)╯彡┻━━┻
```


```suggestion
                'opensuse', 'opensuse-tumbleweed', 'opensuse-leap',
                'sles', 'sle_hpc'):
```